docs/using-hive.md

```diff
@@ -276,6 +328,30 @@ gcp:
   region: us-east1
 ```
 
+For oVirt, replace the contents of `spec.platform` with:
+```yaml
+ovirt:
+  certificatesSecretRef:
+    name: mycluster-ovirt-certs
+  credentialsSecretRef:
+    name: mycluster-ovirt-creds
+  ovirt_cluster_id: 00000000-ovirt-uuid
+  ovirt_network_name: ovirt-network-name
+  storage_domain_id: 00000000-storage-domain-uuid
+```
+
+And create a Secret that holds the CA certificate data for the oVirt environment:
+```yaml
+- apiVersion: v1
+  data:
+    .cacert: base64-encoded-cacert-data-here
```

Comment:
Would it be easier to tell the user to use
```
oc create secret generic my-cluster-ovirt-certs -n mynamespace --from-file=.cacert=$CERT_FILENAME
```
rather than telling them to base64-encoded their cacert?